apps/web/src/components/Inspector.module.css

```diff
@@ -55,17 +56,88 @@
   flex-direction: column;
   min-width: 20rem;
   max-width: 30rem;
-  background-color: #343028;
+  background-color: hsl(40, 13%, 18%);
   overflow-y: auto;
 }
 
 .panel {
   display: flex;
+  flex-direction: column;
   width: 100vw;
   height: 40vh;
   position: fixed;
   bottom: 0;
   left: 0;
   z-index: 2;
   border-top: 1px solid black;
+  background-color: hsl(40, 13%, 8%);
+  color: white;
+
+  button {
+    color: white;
+  }
+}
+
+.sourceViewer {
+  display: flex;
+}
+
+.flagSet {
+  display: flex;
+  flex-direction: row;
```

Comment:
Looks like we could remove this line, since this is a default value for the `flex-direction`.